common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -296,10 +298,8 @@ protected void finalize() throws Throwable {
     }
 
     private static HashedWheelBucket[] createWheel(int ticksPerWheel) {
-        if (ticksPerWheel <= 0) {
-            throw new IllegalArgumentException(
-                    "ticksPerWheel must be greater than 0: " + ticksPerWheel);
-        }
+        checkPositive(ticksPerWheel, "ticksPerWheel");
+
         if (ticksPerWheel > 1073741824) {
```

Comment:
@boris-unckel couldn't we also merge this one with the above if we would use `checkInRange` ?